src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -64,7 +84,18 @@ main() {
     filtered_inputs=()
 
     @@RUN_INPUTS@@
-    nextflow -trace nextflow.plugin $nf_advanced_opts -log ${LOG_NAME} run @@RESOURCES_SUBPATH@@ @@PROFILE_ARG@@ -name run-${NXF_UUID} $nf_run_args_and_pipeline_params "${filtered_inputs[@]}"
+    nextflow -trace nextflow.plugin $nf_advanced_opts -log ${LOG_NAME} run @@RESOURCES_SUBPATH@@ @@PROFILE_ARG@@ -name run-${NXF_UUID} $nf_run_args_and_pipeline_params "${filtered_inputs[@]}" & NXF_EXEC_PID=$!
```

Comment:
Should we use nextflow "${TRACE_CMD}" instead of hardcoding the nextflow.plugin?

Reply:
sorry it got overwritten when I merge the dev. Fixed. Thanks Marek!